### DISCUSSION TITLE
voidable attributes with 0...* cardinality

### DISCUSSION THREAD

**Discussion by bfrichet3** [INSPIRE-MIF/helpdesk#109] [Q&A] [answered]
Dear

I have a question regarding voidable attributes with a cardinality  0...* (see image below). As I understand INSPIRE data specifications if such an attribute is not relevant or does not exist in the context of that dataset, we shouldn't add it. 

For instance we don't have any condominiums. Do we hav to maintain that attribute or to remove though it is a voidable one?

Regards

![image](https://user-images.githubusercontent.com/80756452/176693078-2ac7c4d4-985b-4651-a2fb-d72081087ded.png)

**Comment by fabiovinci** [comment, marked as answer]
Daer @bfrichet3,

as indicated in the TG (I report below an extraction just for convenience), since the "`condominium`" association has multiplicity _0..*_ and is _voidable_ you have two possibilities for its encoding:

1. If you know that no value exists for your AU dataset, you can avoid inserting it.
2. If a value may exist but you don't know, you can insert the attribute with a _void_ value and a related _VoidReasonValue_


![image](https://user-images.githubusercontent.com/22212406/177287274-8543e132-8729-46de-8d16-6e9322baddfa.png)

**Comment by bfrichet3** [comment]
dear @fabiovinci : thank you very much for your confirmation
